### PR TITLE
InfluxDB, charting and a few fixes to Ansible

### PR DESCRIPTION
**Pull request by msbone**
Added InfluxDB for storing timeseries data and created charts using chart.js
Made ansible create a simple environment without any manual work.
Changes done to snmpfetch to store mac/arp tables 

**TODO**

- Currently you can't secure influx and the everyone can run any queries. Also the ansible script do not configure permissions, so write is open to public.

- InfluxDB stores all data from snmpfetch, also text. This is fine but creates some issues when using different vendors. As a field first registered as an int, can't store a string. But can't see any reasons for influx to store strings as of now, so probably a easy fix.

### CODE REVIEW

File: ansible/roles/web/files/gondul.conf [hunk 1]
```diff
@@ -27,6 +27,9 @@
 		Require all granted
         </Directory>
 
+        ProxyPass "/query" "http://localhost:8086/query"
+        ProxyPassReverse "/query" "http://localhost:8086/query"
+
```
Comment (KristianLyng) [inline ansible/roles/web/files/gondul.conf]: ProxyPass ?

We don't use Apache for proxying, so this should be done in Varnish, but I can fix that post-merge...

File: ansible/roles/web/tasks/main.yml [hunk 2]
```diff
@@ -30,15 +30,36 @@
...
     name: cgid
   notify: restart apache
+- apache2_module:
+    state: present
+    name: proxy
+  notify: restart apache
+- apache2_module:
+    state: present
+    name: proxy_http
+  notify: restart apache
```
Comment (KristianLyng) [inline ansible/roles/web/tasks/main.yml]: Ok so there's even more proxy stuff....

**Comment by KristianLyng** [comment]
I think this PR makes it obvious that we need to split up collector gathering from storage, specially in SNMP.

We can't keep refactoring the snmp collector every time we want to test a new db. And the complexity is getting weirdly specific too.

I'll merge this now, since I think it's mostly good, and it solves a specific problem. And we'll see what happens with SNMP in the future. I think we'll probably see REST endpoints for submitting SNMP data.

**Comment by msbone** [comment]
Yeah, the Apache part is just because Varnish was/is currently not in the Ansible job. No reason this can not be done. (Just plain http requests) Happy to create a new merge request if you want.

Have also done a little doodling around the security issue. and concluded that a good way to solve the wide open is to set the password in the "proxypass/whatever" and not from the client.
The other half of the issues is the public/private data, currently InfluxDB do not support "view" or other ways to lock down a request. One way is to duplicate the data in two databases. One "snmp_public" for all requests from public users and one "snmp" for other requests.

SNMP fetching is also a nice story for itself, but not what this was trying to solve :) But happy to help

**Pull request merged**